src/components/Containers/UASLayout/UASLayout.jsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+
+import { Text } from "../../Texts";
+import { baseTheme } from "../../../theme";
+import { media } from "../../../utils";
+
+export const UASLayout = ({ titleText, graphic, descriptionText, ...props }) => (
+  <div {...props}>
+    <TitleSection>
+      <Title size={baseTheme.size.h2} bold="true">
+        {titleText}
+      </Title>
+    </TitleSection>
+    <InfoSection>
+      <Graphic src={graphic} />
+      <Description>{descriptionText}</Description>
+    </InfoSection>
+  </div>
+);
+
+const Title = styled(Text)`
+  ${({ theme }) => `
+      font-family: ${theme.font.josefin};
+      color: ${theme.colors.black};
+  `};
+`;
+
+const Description = styled(Text)`
```

Comment:
```Text``` is a ```p``` element and by default there is a ```margin-block-start``` of 1em. If you want it to be perfectly aligned, then a ```margin: 0``` on desktop (idk about other platforms) would work. See below:

<img width="898" alt="Screen Shot 2021-07-19 at 3 52 12 PM" src="https://user-images.githubusercontent.com/62865150/126219108-c54cf9d6-c73a-451d-b382-bcd43cc5fd12.png">

However, on the whole I think this approach might be a little risky if there is white space surrounding the ```svg``` image so I would recommend the ```align-items: center``` property on the ```InfoSection``` element, but this is up to your discretion.

Reply:
This way the image and text are both centered ^^